hr_recruitment_phone_log/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==========================
+Show phonecalls on partner
+==========================
+
+This module restores the phonecall list on the partner view.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/connector-telephony/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/connector-telephony/issues/new?body=module:%20partner_phone_log%0Aversion:%209.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
On 08/25/2016 05:56 PM, Eric Caudal wrote:
>
> In hr_recruitment_phone_log/README.rst
> <https://github.com/OCA/connector-telephony/pull/101#discussion_r76346020>:
>
> > +Show phonecalls on partner
> > +==========================
> > +
> > +This module restores the phonecall list on the partner view.
> > +
> > +.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
> > +   :alt: Try me on Runbot
> > +   :target: https://runbot.odoo-community.org/runbot/140/9.0
> > +
> > +Bug Tracker
> > +===========
> > +
> > +Bugs are tracked on `GitHub Issues <https://github.com/OCA/connector-telephony/issues>`_.
> > +In case of trouble, please check there if your issue has already been reported.
> > +If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
> > +`here <https://github.com/OCA/connector-telephony/issues/new?body=module:%20partner_phone_log%0Aversion:%209.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
>
> remove
>
I am sorry, remove what, the file, the module? I am working to make the
changes you have requested.